keylime/ima/ast.py

```diff
@@ -269,8 +269,9 @@ def create_signature(hexstring: str) -> Optional[Signature]:
         """Create the Signature object if the hexstring is a valid signature"""
         try:
             return Signature(hexstring)
-        except ParserError:
-            pass
+        except ParserError as e:
+            if hexstring:
```

Comment:
I agree on not hiding the exception under the pass, but in that case I am not sure why to hide it if the signature is empty or None

Reply:
This commit has already been applied and shouldn't be part of this PR.